chart/templates/workers/worker-kedaautoscaler.yaml

```diff
@@ -36,13 +36,13 @@ metadata:
 spec:
   scaleTargetRef:
     deploymentName: {{ .Release.Name }}-worker
-  pollingInterval:  {{  .Values.workers.keda.pollingInterval }}   # Optional. Default: 30 seconds
-  cooldownPeriod: {{  .Values.workers.keda.cooldownPeriod }}    # Optional. Default: 300 seconds
-  maxReplicaCount: {{  .Values.workers.keda.maxReplicaCount }}   # Optional. Default: 100
+  pollingInterval:  {{ .Values.workers.keda.pollingInterval }}   # Optional. Default: 30 seconds
+  cooldownPeriod: {{ .Values.workers.keda.cooldownPeriod }}    # Optional. Default: 300 seconds
+  maxReplicaCount: {{ .Values.workers.keda.maxReplicaCount }}   # Optional. Default: 100
   triggers:
     - type: postgresql
       metadata:
         targetQueryValue: "1"
         connection: AIRFLOW_CONN_AIRFLOW_DB
-        query: "SELECT ceil(COUNT(*)::decimal / 16) FROM task_instance WHERE state='running' OR state='queued'"
+        query: "SELECT ceil(COUNT(*)::decimal / {{ .Values.config.celery.worker_concurrency }}) FROM task_instance WHERE state='running' OR state='queued'"
```

Comment:
i wonder if a better solution would be to allow user to specify the entire query, instead of `worker_concurrency`.  for one, this would resolve the problem highlighted in my other comment.  additionally though it would open the door to support for mysql.  mysql has a different query syntax but otherwise there is a keda trigger for it and we could add support for it by enabling custom query.  custom query is also more flexible in general.

Reply:
Currently the chart does not support MySQL anyway. Agree in principle though

Reply:
the chart does not support mysql when you run the metastore on k8s along with airflow.  however, wouldn't it work with mysql if you had an external metastore e.g. rds (which is what is recommended anyway)?  

i know postgres is the better fit for airflow but some folks might use other dbs and off top of my head i think it would probably work since you just need to specify conn uri and disable postgres and pgbouncer...

Reply:
oh yes -- you are right